.github/workflows/lint.yml

```diff
@@ -37,24 +37,24 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install Node.js dependencies
-        run: npm ci
+      - name: Install Cypress
+        run: npm i cypress
 
-      - name: Build App
-        run:  npm run build
+      - name: Build and serve App Container
+        run: docker-compose -f "docker-compose.ci.yml" up -d --build 
 
       - name: Run linters
         uses: wearerequired/lint-action@v2
         with:
           eslint: true
           eslint_extensions: "js,jsx,ts,tsx"
+      
+      - name: Run Jest tests
+        run: docker exec dev-directory_app_1 npm run test
```

Comment:
does this run tests against the built files, or is it just running in the context of the app?

Reply:
Have corrected this.